src/Build.UnitTests/BackEnd/TaskBuilder_Tests.cs

```diff
@@ -1238,14 +1238,11 @@ public string Name
             }
 
             /// <summary>
-            /// Returns the build parameters.
+            /// Gets or sets the build parameters.
             /// </summary>
             public BuildParameters BuildParameters
             {
-                get
-                {
-                    return _buildParameters;
-                }
+                get => _buildParameters; set => throw new NotSupportedException();
```

Comment:
This change doesn't seem necessary?

Reply:
`MockHost` implements `IBuildComponentHost`, which requires a setter.